react-ui/package.json

```diff
@@ -12,6 +12,7 @@
     "node-sass": "^4.14.1",
     "query-string": "^6.13.6",
     "react": "^16.13.1",
+    "react-confirm-alert": "^2.6.2",
```

Comment:
Please note that a new package was installed so make sure to `npm install` inside the `react-ui` folder to properly run and test the code locally.

Reply:
Thank you for the heads up - will do that!